src/modules/endpoint/fetchers/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as categoriesFetcher } from './categories'
```

Comment:
I don't think we need a file to import and export files which are in the corrosponing directory. Does this make things easier?

Reply:
not really